File: Programming_Basics_with_Python/exam_prep/02._add_bags.py
price_over_20_kg = float(input())
kg = float(input())
days = int(input())
count = int(input())
price_per_one = 0
if kg < 10:
    price_per_one = 0.2 * price_over_20_kg
elif 10 <= kg <= 20:
    price_per_one = 0.5 * price_over_20_kg
elif kg > 20:
    price_per_one = price_over_20_kg
if days > 30:
    price_per_one = price_per_one * 1.1
elif 7 <= days <= 30:
    price_per_one = price_per_one * 1.15
elif days < 7:
    price_per_one = price_per_one * 1.4
total_price = count * price_per_one
print(f" The total price of bags is: {total_price:.2f} lv. ")

File: Programing_Basics_with_JS/examPreparation/programmingBasicsOnlineExam18And19July2020/02.AddBags.js
function addBags(input) {
    let luggagePriceMoreOf20kg = Number(input[0]);
    let luggageKilograms = Number(input[1]);
    let daysToGoAway = Number(input[2]);
    let luggageCount = Number(input[3]);

    let luggagePriceTo10kg = luggagePriceMoreOf20kg * 0.2;
    let luggagePriceTo20kg = luggagePriceMoreOf20kg * 0.5;
    let finalLuggagePrice = 0;

    if(luggageKilograms < 10) {
        finalLuggagePrice = luggagePriceTo10kg;
    } else if (luggageKilograms >= 10 && luggageKilograms <= 20) {
        finalLuggagePrice = luggagePriceTo20kg;
    } else if (luggageKilograms > 20) {
        finalLuggagePrice = luggagePriceMoreOf20kg;
    }

    if(daysToGoAway > 30) {
        finalLuggagePrice += finalLuggagePrice * 0.1;
    } else if (daysToGoAway >= 7 && daysToGoAway <= 30) {
        finalLuggagePrice += finalLuggagePrice * 0.15;
    } else if (daysToGoAway < 7) {
        finalLuggagePrice += finalLuggagePrice * 0.4;
    }

    finalLuggagePrice = finalLuggagePrice * luggageCount;

    console.log(`The total price of bags is: ${finalLuggagePrice.toFixed(2)} lv.`);
}

addBags(["30",
"18",
"15",
"2"])

addBags(["25.50",
"5",
"36",
"6"])

addBags(["63.80",
"23",
"3",
"1"])

